Ghidra/Debug/Debugger/src/main/java/ghidra/app/plugin/core/debug/gui/interpreters/AbstractDebuggerWrappedConsoleConnection.java

```diff
@@ -69,14 +69,8 @@ public void consoleOutput(TargetObject console, Channel channel, byte[] out) {
 			if (os == null) {
 				return;
 			}
-			/**
-			 * NB: yes, the extra space is lame... The InterpreterPanel's repositionScrollPane
-			 * method subtracts 1 from the text length to compute the new position causing it to
-			 * scroll to the last character printed. We want it to scroll to the next line, so...
-			 */
 			try {
 				os.write(out);
-				os.write(' ');
```

Comment:
I'm all for removing this code, if it's actually no longer needed. I see a corresponding change in `InterpreterPanel.java` below.... Good, but that might be there for a reason.